pkgs/top-level/all-packages.nix

```diff
@@ -13723,7 +13709,8 @@ let
     retroarch = retroarchBare;
   });
 
-  rssglx = callPackage ../misc/screensavers/rss-glx { };
+  rss-glx = callPackage ../misc/screensavers/rss-glx { };
+  rssglx = builtins.trace "WARNING: rssglx package was renamed to rss-glx" pkgs.rss-glx;
```

Comment:
I think it's worth it, but you should add the backwards compatibility attribute to the bottom of all-packages.nix.

However, the warning message should be omitted, because it will be triggered by `nix-env -qa`.